torchmdnet/scripts/train.py

```diff
@@ -63,6 +63,7 @@ def get_args():
     parser.add_argument('--model', type=str, default='graph-network', choices=models.__all__, help='Which model to train')
     parser.add_argument('--output-model', type=str, default='Scalar', choices=output_modules.__all__, help='The type of output model')
     parser.add_argument('--prior-model', type=str, default=None, choices=priors.__all__, help='Which prior model to use')
+    parser.add_argument('--prior-args', default=None, type=str, help='Additional arguments for the prior model. Need to be specified in JSON format i.e. \'{"cutoff_distance": 10.0, "max_num_neighbors": 100}\'')
```

Comment:
how does this work? I don't see the json being converted to a dictionary anywhere. Also how would it work from a yaml config file? could you provide some examples?

Reply:
Although people mostly use yaml for simple config files, it actually is a strict superset of json.  So there's nothing to convert: json syntax is yaml syntax, and the dict gets parsed automatically.